src/app/home/home.component.ts

```diff
@@ -16,18 +19,95 @@ export class HomeComponent implements OnInit {
 
   /** Username of authenticated user. */
   username: string;
+  /** Activity Form. */
+  activityForm: any;
+  /** Search Text. */
+  searchText: FormControl = new FormControl();
+  /** Filtered Activities. */
+  filteredActivities: Observable<any[]>;
+  /** All User Activities. */
+  allActivities: any[] = [
+    { activity: 'client', path: '/clients'},
+    { activity: 'groups', path: '/groups'},
+    { activity: 'centers', path: '/centers'},
+    { activity: 'accounting', path: '/accounting'},
+    { activity: 'users', path: '/users'},
+    { activity: 'organization', path: '/organization'},
+    { activity: 'system', path: '/system'},
+    { activity: 'templates', path: '/templates'},
+    { activity: 'self-service users', path: '/self-service/users'},
+    { activity: 'self-service app-configuration', path: '/self-service/app-configuration'},
+    { activity: 'task management', path: '/self-service/task-management'},
+    { activity: 'create group', path: ''},
+    { activity: 'create center', path: '/centers/create'},
+    { activity: 'configuration', path: '/system/global-configurations'},
+    { activity: 'tasks', path: '/self-service/task-management'},
+    { activity: 'create template', path: '/self-service/task-management'},
+    { activity: 'create loan product', path: '/products/loan-products/create'},
+    { activity: 'create saving product', path: '/products/saving-products/create'},
+    { activity: 'roles', path: '/system/roles-and-permissions'},
+    { activity: 'add role', path: '/system/roles-and-permissions/add'},
+    { activity: 'configure maker checker tasks', path: '/system/global-configurations'},
+    { activity: 'loan products', path: 'products/loan-products'},
+    { activity: 'charges', path: 'products/charges'},
+    { activity: 'saving products', path: 'products/saving-products'},
+    { activity: 'offices', path: 'organization/offices'},
+    { activity: 'create office', path: 'organization/offices/create'},
+    { activity: 'currency configurations', path: 'organization/currencies'},
+    { activity: 'user settings', path: ''},
+    { activity: 'employees', path: 'organization/employees'},
+    { activity: 'create employee', path: 'organization/employees/create'},
```

Comment:
you have forgotten to add '/' at start of route

Reply:
yes. I'll fix it. But can you guide me to implement the mat autocomplete as in the app? I did not find any similar components to this other than filtering used for mat tables